src/EasyList/TodoMenu.cs

```diff
@@ -1,101 +1,100 @@
 ﻿using EasyList.DataModels;
 using EasyList.Enums;
-using EasyList.Factories;
-using EasyList.Interfaces;
 using Sharprompt;
 using System;
 using System.Collections.Generic;
-using System.Linq;
 
 namespace EasyList
 {
     public class TodoMenu
     {
+
         public static void Run()
         {
-
-            while(true)
+            var _todoValidate = new Validate();
+            while (true)
             {
                 var action = Prompt.Select<TODOMENU>("Welcome to EasyList!");
                 //Refactor this such that adding new should not modify this input layer
                 switch (action)
```

Comment:
And if statement would be a bit cleaner. Especially when you have logic in every case.

Reply:
What do you think about this ?
![image](https://user-images.githubusercontent.com/63421973/130258708-fe6f719e-96af-4eba-a6fd-f261aedf90f5.png)